.travis.yml

```diff
@@ -35,11 +35,18 @@ addons:
       - python
       - valgrind
       - sparse
+      - wget
 
```

Comment:
I prefer the command line of wget. It is by default installed in travis image. I added wget for docker. So technically it does not add extra time.